tests/testthat/test-recall_data.R

```diff
@@ -0,0 +1,206 @@
+test_that("recall_by_make returns the correct shape",{
```

Comment:
can this test just be merged into the next suite? anything based on adding the `year` args should also pass for this one

Reply:
merged the below test into this one